File: solutions/c/raindrops/1/raindrops.c
#include "raindrops.h"

void convert(char result[], int drops){
    if (drops%3==0){
        sprintf(result+strlen(result),"Pling");
    }
    if (drops%5==0){
        sprintf(result+strlen(result),"Plang");
    }
    if (drops%7==0){
        sprintf(result+strlen(result),"Plong");
    }
    if (drops%3!=0 && drops%5!=0 && drops%7!=0){
        sprintf(result,"%d",drops);
    }
}

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import(
    "fmt"
)

func Convert(number int) string {
	answer := ""
    if number%3==0{
        answer = answer + "Pling"
    }
    if number%5==0{
        answer = answer + "Plang"
    }
    if number%7==0{
        answer += "Plong"
    }
    if number%3!=0 && number%5!=0 && number%7!=0{
        return fmt.Sprintf("%v",number)
    }
    return answer
}
